src/SDKs/HDInsight/Management.HDInsight/Customizations/Models/AzureStorageInfo.cs

```diff
@@ -57,13 +57,5 @@ public AzureStorageInfo(string storageAccountName, string storageAccountKey, str
             this._storageAccountKey = storageAccountKey;
             this._storageContainer = storageContainer;
         }
-
-        public string StorageAccountUri 
```

Comment:
This property isn't used anywhere, and is likely an artifact of legacy code.

Reply:
This is a breaking change and needs the package version to be set to `3.1.0-preview`